### PR TITLE
fix: Modal.method bodyStyle not working

### LINKED ISSUE #32910: 使用 Modal.method  bodyStyle无效
### 重现链接

https://codesandbox.io/s/xin-xi-ti-shi-tdg86?file=/index.js

### antd 版本

>4.8.0

### 重现步骤

1.点击info按钮
2.模态框bodyStyle padding属性未生效

### 期望的结果是什么？

样式正常带入到modal的body中

### 实际的结果是什么？

bodyStyle无任何效果

### 浏览器

Chrome

### 浏览器版本

_No response_

### 操作系统

Windows

### PR DESCRIPTION
**Pull request by jin-benben**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close #32910

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Fix Modal.method `bodyStyle` not working.       |
| 🇨🇳 Chinese |  修复 Modal.method `bodyStyle` 无效的问题。         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
来个测试用例

**Comment by jin-benben** [comment]
> 来个测试用例



> 来个测试用例

不好意思，以前没怎么写过测试用例。该怎么写？

**Comment by jin-benben** [comment]
> 来个测试用例

测试用例已补充。

File: components/modal/__tests__/confirm.test.js [hunk 1]
```diff
@@ -545,4 +545,10 @@ describe('Modal.confirm triggers callbacks correctly', () => {
     await sleep(500);
     expect(afterClose).toHaveBeenCalled();
   });
+
+  it('bodyStyle', async () => {
+    open({ bodyStyle: {width:500} });
```
Comment (afc163) [inline components/modal/__tests__/confirm.test.js:550]:
```suggestion
    open({ bodyStyle: { width: 500 }});
```

**Pull request merged**